src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIMetadataService.java

```diff
@@ -115,23 +200,78 @@ public String deleteMetadataTemplate(Long templateId, Long projectId, Locale loc
 	 * @param templateId Identifier for a {@link MetadataTemplate}
 	 * @param projectId Identifier for a {@link Project}
 	 * @param locale     Current users {@link Locale}
-       * @return text to display to user about the result of updating the default metadata template
+	 * @return text to display to user about the result of updating the default metadata template
 	 * @throws Exception if there is an error updating the default metadata template for a project
 	 */
 	public String setDefaultMetadataTemplate(Long templateId, Long projectId, Locale locale) throws Exception {
```

Comment:
I think this also needs to be annotated with `@Transactional` as we are updating the metadataTemplate on line 212 which is read outside of a Transaction and that has an attribute Project which would get loaded for dirty checking etc.

Reply:
Added in c427089ae2f1a86e46a20ce76e32f226b2ca3b89